packages/firestore/src/api/user_data_reader.ts

```diff
@@ -138,11 +138,14 @@ interface ContextSettings {
    * A path within the object being parsed. This could be an empty path (in
    * which case the context represents the root of the data being parsed), or a
    * nonempty path (indicating the context represents a nested location within
-   * the data).
+   * the data). Defaults to null.
```

Comment:
Doesn't it default to "undefined"? Consider just omitting this statement because it is seems like it is just explaining how the "?" property modifier works.

Reply:
That's true. I dropped it.